core/ir/opnd_shared.c

```diff
@@ -743,7 +743,85 @@ opnd_create_base_disp_aarch64(reg_id_t base_reg, reg_id_t index_reg,
         CLIENT_ASSERT(false, "opnd_create_base_disp_aarch64: invalid extend type");
     return opnd;
 }
-#endif
+
+opnd_t
+opnd_create_cond(int cond)
+{
+    /* FIXME i#1569: Move definition of dr_pred_type_t to opnd.h for AArch64
```

Comment:
Please create an issue separate from the AArch64 master issue #1569 to track this.

Reply:
Created: #4502.